policy/README.opa.md

```diff
@@ -47,7 +47,7 @@ scheme-specific.
 `result` is a JSON object representing `proto.AttestationResult` that was
 generated by the scheme.
 
-`format` is the name of `proto.AttestationFormat` associated with the scheme.
+`scheme` is the name of the attestation scheme.
```

Comment:
All looks good here,  while you are making this change, shall we be more precise and change Line 40: to say:
`evidence` contains the JSON representation of `proto.EvidenceContext.Evidence`